File: Iniciante/URI 1009.cpp
#include <iostream>
 
int main() { 
    char nome;
    double salarioFixo, vendas, total;

    scanf("%s %lf %lf", &nome, &salarioFixo, &vendas);
    total = ((vendas * 15) / 100) + salarioFixo;
    printf("TOTAL = R$ %.2f\n", total);

    return 0;
}

File: Iniciante/URI 1009.go
package main
 
import (
    "fmt"
)
 
func main() {
    var nome string
    var salarioFixo, vendas, total float64

    fmt.Scanf("%s", &nome)
    fmt.Scanf("%f", &salarioFixo)
    fmt.Scanf("%f", &vendas)

    total = ((vendas * 15) / 100) + salarioFixo

    fmt.Printf("TOTAL = R$ %.2f\n", total)
}